app/helpers/navigation_helper.rb

```diff
@@ -26,7 +26,27 @@ def show_bottom_navbar?
       !current_page?(coming_soon_path)
   end
 
+  def show_back_btn?
```

Comment:
Prefer to use full word
```suggestion
  def show_back_button?
```